src/CMakeLists.txt

```diff
@@ -308,36 +301,33 @@ configure_file(
 install(FILES ${GENERATED_DIR}/rdkafka.pc
   DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig"
 )
-if(RDKAFKA_BUILD_STATIC)
```

Comment:
Is this left off because, in theory, rdkafka.pc == rdkafka-static.pc?

Reply:
It's been a while. Not sure why I removed the section. I have now restored it and all spotted a few little mistakes. Now fixed.